### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by igor-stasiv**
- [DEMO LINK](https://igor-stasiv.github.io/layout_human-rights/) <br>
- [TEST REPORT LINK](https://igor-stasiv.github.io/layout_human-rights/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,14 +1,172 @@
...
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <h1>Universal Declaration of Human Rights</h1>
+
+    <h2>Preamble</h2>
+    <p>Whereas recognition of the inherent dignity and of the equal and inalienable
+    rights of all members of the human family is the foundation of freedom, justice
+    and peace in the world,
+      <br>
```
Comment (vsm2k17) [inline src/index.html:18]:
```suggestion
    <p>
      Whereas recognition of the inherent dignity and of the equal and inalienable
      rights of all members of the human family is the foundation of freedom, justice
      and peace in the world,
      <br>
```

Comment (vsm2k17) [reply src/index.html:18]: in that way looks better, and easier to read

Comment (vsm2k17) [reply src/index.html:18]: all content of any tag should have 2 indents more than the tag itself
(or 4 if you use 4 indents everywhere, in MA we use 2 usually but it is optional)

Comment (vsm2k17) [reply src/index.html:18]: fix indent issue everywhere

**Review by mykola-yarchuk — CHANGES_REQUESTED** [review]

**Review by mykola-yarchuk — APPROVED** [review]
good job

**Pull request closed without merging**